docs/en/writing-running-appium/mobile-web.md

```diff
@@ -159,7 +159,7 @@ class ContextTests extends PHPUnit_Extensions_AppiumTestCase
 Pre-requisites:
 
 *  Make sure Chrome (an app with the package `com.android.chrome`) is installed on your device or emulator. Getting Chrome for the x86 version of the emulator is not currently possible without building Chromium, so you may want to run an ARM emulator and then copy a Chrome APK from a real device to get Chrome on an emulator.
-*  If downloaded from [NPM](https://www.npmjs.org/package/appium), or running from the [.app](https://github.com/appium/appium-dot-app), nothing needs to be done. If running from source, `npm install` will download ChromeDriver and put it in `node_modules/appium-chromedriver/chromedriver/<OS name>/` for users having npm v3+ and for npm v2 it will be in `node_modules/appium-android-driver/node_modules/appium-chromedriver/chromedriver/<OS name>/`. A particular version can be specified by passing the `--chromedriver_version` config property (e.g., `npm install appium --chromedriver_version="2.16"`), otherwise the most recent one will be retrieved.
+*  Chromedriver needs to be installed and capable of automating the Chrome on the device. See [here](/docs/en/advanced-concepts/chromedriver.md).
```

Comment:
I would expand this sentence a bit - to me, the current structure of this sentence does not imply that there may be additional steps necessary with respect to configuring Chromedriver.

Perhaps instad of "... and capable of automating the Chrome on the device. See here." something like "... and configured for automating the specific version of Chrome available on the device. See here for more information and details."